### PR TITLE
SAIC-181: Extend devlab to have more compute node

### PR DESCRIPTION
**Pull request by plomakin**
Added icehouse and grizzly's compute nodes to Vagrantfile.
Updated configs for devlab using computes.
Updated repackage_box with to generate new boxes.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

File: devlab/utils/repackage_box.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ ssh_host=$(echo "$ssh_config" | grep HostName | cut -d' ' -f4)
 ssh_port=$(echo "$ssh_config" | grep Port | cut -d' ' -f4)
 ssh_private_key=$(echo "$ssh_config" | grep IdentityFile | cut -d' ' -f4)
 ssh_user=$(echo "$ssh_config" | grep '\<User\>' | cut -d' ' -f4)
-ssh_users_private_key=~/.ssh/id_rsa.pub
+ssh_users_private_key=~/.vagrant.d/insecure_private_key
```
Comment (roman-verchikov) [inline devlab/utils/repackage_box.sh]: this is not used anywhere, lets remove it

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by roman-verchikov** [comment]
+1

**Pull request closed without merging**